doc/site/en/guides/border-router/beaglebone-black.md

```diff
@@ -1,338 +1,204 @@
 
-> Note: This example image has a single partition,
-  other images may have additional partitions.
+1. Download the latest Debian IoT image for BeagleBone. [beaglebone latest
+   images](https://beagleboard.org/latest-images)
+   *  The version used for this guide was
+      `bone-debian-10.3-iot-armhf-2020-04-06-4gb.img.xz`
+1. Install the OS image on a uSD Card by following the [BeagleBone getting
+   started guide](https://beagleboard.org/getting-started).
+1. Boot the BeagleBone and SSH into the device.
+   *  Connectivity over a local Ethernet based network is recommended.
+   *  The cloud9 IDE will be disabled later in this guide.
+   *  This guide will change the state of BeagleBone network interfaces, be
+      aware you secure shell may disconnect.
+   *  Modern BeagleBone bootloaders will run from the uSD card by default, but
+      some BeagleBone Black devices may try to boot from the internal eMMC.
+      Make sure to press the BOOT Button in this case.
 
-### 2. Create the new, larger partition
+Warning: The power requirements of the development kit used for the OpenThread
+RCP may be too great for the power that can be supplied from a computer's USB
+port. It is recommended that you use the 5V power adaptor for the BeagleBone
+where applicable.
 
-Run `fdisk` on the device (SD card):
+For more detailed information on the BeagleBone, see the [BeagleBoard Support
+Page](https://beagleboard.org/support).
 
-```
-root@beaglebone:/home/debian# fdisk /dev/mmcblk0
-
-Welcome to fdisk (util-linux 2.25.2).
-Changes will remain in memory only, until you decide to write them.
-Be careful before using the write command.
-```
+## Step 2: Prepare the Debian Environment for OTBR
 
-Print the current partition table to find the starting sector.
-The value of the `Start` field is the starting sector for the target
-partition. It should be listed with the same partition name as in Step 1,
-with a `Type` of `Linux`. In the output below, the starting sector is `8192`.
+Certain parts of the default BeagleBone Debian image run by default. These may
+conflict with some parts of the OpenThread Border Router software.
 
-```
-Command (m for help): p
-Disk /dev/mmcblk0: 7.2 GiB, 7744782336 bytes, 15126528 sectors
-Units: sectors of 1 * 512 = 512 bytes
-Sector size (logical/physical): 512 bytes / 512 bytes
-I/O size (minimum/optimal): 512 bytes / 512 bytes
-Disklabel type: dos
-Disk identifier: 0xca52207f
-
-Device         Boot Start     End Sectors  Size Id Type
-/dev/mmcblk0p1 *     8192 6963199 6955008  3.3G 83 Linux
-```
+Some packages are running by default on the BeagleBone to enable quick
+development. These can be found in systemd with the command `sudo systemctl
+list-units --all` and `sudo systemctl list-sockets --all`.
 
-Delete the existing partition:
+Stop and disable the modules:
 
 ```
-Command (m for help): d
-Selected partition 1
-Partition 1 has been deleted.
+sudo systemctl stop bonescript-autorun.service
```

Comment:
Can you add $ in front of any commands in code blocks throughout the doc so it's clear it's a command and not output

Reply:
done, I assume you don't have a convention of which one is supposed to have elevated privileges. I just put `sudo` in-front of anything that needs super-user.

Reply:
Yeah `sudo` in front of everything is fine, thanks